src/modules/chat/tokenizers.jsx

```diff
@@ -433,6 +433,13 @@ export const UserHighlights = {
 			}
 		}
 
+		if(this.context.manager.provider.get('chat.filtering.highlight-temp')?.includes(msg.user.userID))
```

Comment:
Careful, this is an incorrect use of the optional chaining operator. This could throw an exception. You should have `?.includes?.(...)` for safety.

More reading: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Easiest thing to do here would be to use the default value for `.get()`, of course.

Reply:
My incompetence with JS is showing 😂 I'll fix it in the addon PR (I will remove this once I'm done)